Sources/ClientRuntime/Networking/Http/Middlewares/SignerMiddleware.swift

```diff
@@ -69,10 +70,17 @@ extension SignerMiddleware: ApplySigner {
             )
         }
 
+        // Check if CRT should be provided a pre-computed Sha256 SignedBodyValue
+        var updatedSigningProperties = signingProperties
+        let sha256: String? = attributes.get(key: AttributeKey(name: "X-Amz-Content-Sha256"))
+        if let bodyValue = sha256 {
+            updatedSigningProperties.set(key: AttributeKey(name: "SignedBodyValue"), value: sha256)
+        }
```

Comment:
SHA256Middleware if executed will set this attribute to be picked up by the SignerMiddleware and put into the signing properties